### DISCUSSION TITLE
how to get lazy load audio ?

### DISCUSSION THREAD

**Discussion by odaymusic** [vercel/community#984] [Help] [answered]
i build audio store but i need method for lazy load audio list

**Comment by amyegan** [comment, marked as answer]
Hi @odaymusic! The most straightforward solution I can think of is using the native [`audio` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio) with [`preload="none"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#attr-preload) to prevent preloading audio.
